src/internet_identity/src/storage/tests.rs

```diff
@@ -29,16 +29,16 @@ fn should_report_max_number_of_entries_for_256gb() {
 }
 
 #[test]
-fn should_serialize_header_v8() {
+fn should_serialize_header_v9() {
```

Comment:
nit: shouldn't we still keep the test for `v8`?  (i.e. add a new one for `v9`)

Reply:
New storage is now automatically initialized in `v9`. The ability to initialize `v8` is removed.

Compatibility with existing `v8` storage is tested in the (unchanged) `should_recover_header_from_memory_v8` test.